build.gradle

```diff
@@ -365,17 +365,32 @@ task lintTest {
     dependsOn checkstyleTest, pmdTest, findbugsTest
 }
 
-task macker << {
+task mackerMain << {
```

Comment:
I am not expert in `Gradle`. The two tasks are almost the same. Is it possible to combine it?